lib/gitlab/client/markdown.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Gitlab::Client
+  # Defines methods related to markdown.
+  # @see https://docs.gitlab.com/ce/api/avatar.html
```

Comment:
https://docs.gitlab.com/ce/api/markdown.html

Reply:
Sorry. Missed it. Fixed